src/routes/content/[...id].svelte

```diff
@@ -0,0 +1,50 @@
+<script lang="ts">
+	import { goto } from "$app/navigation";
+	import type * as api from "src/api";
+	import { InsertFormType } from "src/api/forms/base";
+	import Content from "src/components/info/Content.svelte";
+	import DeleteModal from "src/components/info/DeleteModal.svelte";
+	import { draftForm } from "src/utils/forms/stores";
+	import { v4 as uuid } from "uuid";
+
+	export let content: api.Content;
+
+	function handleEdit() {
+		const id = uuid();
+		draftForm.createForm(id, [{
+			type: InsertFormType.CONTENT,
+			value: content,
+		}]);
+
+		goto(`/forms/${id}`);
+
+		// TODO: handle updates to existing nodes within insertForm functionality to support editing content
+	}
+
+	let showDelete = false;
+	function handleDelete() {
+		// TODO: create API route for node "delete"
+		alert("Not implemented");
```

Comment:
Probably for the best that delete isn't yet implemented, really; I think as we have it right now, only an admin can completely delete a node. Even when this is implemented, we probably want to leave the history in the revision table, so it can be restored in the event of vandalism.